src/test/java/org/takes/facets/auth/social/PsGoogleTest.java

```diff
@@ -57,8 +57,48 @@
 public final class PsGoogleTest {
 
     /**
+     * Octocat.
+     */
+    private static final String OCTOCAT = "octocat";
```

Comment:
@dalifreire This constant should be different in each test since it is not a fixed google API value. Please see http://www.yegor256.com/2016/05/03/test-methods-must-share-nothing.html for further info.